src/routes/seller.routes.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, Response, Router } from "express";
+import SellerController from "../controllers/seller.controller";
+
+const sellerRouter = Router();
+const sellerController = new SellerController();
+
+sellerRouter.get("/getVerifiedSeller", sellerController.getVerifiedSeller);
+
+sellerRouter.get("/getPendingSeller", sellerController.getPendingSeller);
+
+sellerRouter.patch(
+  "/changeSellerStatus/approvedSeller",
+  (req: Request, res: Response, next: NextFunction) => {
+    req.body.is_verified = true;
+    next();
```

Comment:
Why not make a middleware function rather than using callback function in here ?

Reply:
i Will add middleware in next Commit